clients/docs/c.rst

```diff
@@ -0,0 +1,170 @@
+.. _client-examples-c:
+
+C
+=
+
+
+In this tutorial, you will run a C client application that produces messages to
+and consumes messages from an |ak-tm| cluster.
+
+.. include:: includes/client-example-overview.rst
+
+
+Prerequisites
+-------------
+
+Client
+~~~~~~
+
+-  `librdkafka <https://github.com/edenhill/librdkafka>`__ installed on
+   your machine. See the `librdkafka installation
+   instructions <https://github.com/edenhill/librdkafka/blob/master/README.md#instructions>`__.
+
+-  Create a local file (for example, at ``$HOME/.confluent/librdkafka.config``)
```

Comment:
@tikimims this bullet should be deleted (it's covered by the include in Kafka Cluster section)